content/it/docs/concepts/containers/container-environment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Container Environment
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Questa pagina descrive le risorse disponibili nei Containers eseguiti in Kubernetes.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Container environment
+
+Quando si esegue un Container in Kubernetes, le seguenti risorse sono rese disponibili:
+
+* Un filesystem, composto dal file system dell'[image](/docs/concepts/containers/images/) e da uno o più [volumes](/docs/concepts/storage/volumes/).
+* Una serie di informazioni sul container stesso.
+* Una serie di informazioni sugli oggetti nel cluster.
+
+### Informazioni sul Container
+
+L' *hostname* di un container è il nome del Pod all'interno del quale è eseguito il container.
```

Comment:
On line 9, Container was capitalized, while here it is now. What is the right form?

Reply:
I don't have strong preference here, converging on Capitalize for consistency with the English version